Test/UnitTests/KeyTriggerTest.cs

```diff
@@ -96,6 +96,72 @@ public void KeyTrigger_InvokesActions_WhenKeyIsPressed(Key key)
             }
         }
 
+        [TestMethod]
+        [DataRow(Key.A)]
+        [DataRow(Key.B)]
+        [DataRow(Key.C)]
+        [DataRow(Key.D)]
+        [DataRow(Key.E)]
+        [DataRow(Key.F)]
+        [DataRow(Key.G)]
+        [DataRow(Key.H)]
+        [DataRow(Key.I)]
+        [DataRow(Key.J)]
+        [DataRow(Key.K)]
+        [DataRow(Key.L)]
+        [DataRow(Key.M)]
+        [DataRow(Key.N)]
+        [DataRow(Key.O)]
+        [DataRow(Key.P)]
+        [DataRow(Key.Q)]
+        [DataRow(Key.R)]
+        [DataRow(Key.S)]
+        [DataRow(Key.T)]
+        [DataRow(Key.U)]
+        [DataRow(Key.V)]
+        [DataRow(Key.W)]
+        [DataRow(Key.X)]
+        [DataRow(Key.Y)]
+        [DataRow(Key.Z)]
+        [DataRow(Key.NumPad1)]
+        [DataRow(Key.NumPad2)]
+        [DataRow(Key.NumPad3)]
+        [DataRow(Key.NumPad4)]
+        [DataRow(Key.NumPad5)]
+        [DataRow(Key.NumPad6)]
+        [DataRow(Key.NumPad7)]
+        [DataRow(Key.NumPad8)]
+        [DataRow(Key.NumPad9)]
+        [DataRow(Key.Enter)]
+        [DataRow(Key.Tab)]
+        [DataRow(Key.LeftCtrl)]
```

Comment:
I'm surprised that the Ctrl key down cases don't report Ctrl as modifiers though